src/helpers/ThemeLoader.ts

```diff
@@ -0,0 +1,68 @@
+import IOverrideTheme from "../Types/IOverrideTheme"
+import * as Flex from "@twilio/flex-ui";
+import IManageThemes from "../API/IManageThemes";
+
+export default class ThemeService {
+  constructor(private _apiClient: IManageThemes) {  }
+
+  getAllThemes = async () => {
+    return await this._apiClient.getAllThemesAsync();
+  };
+
+  getUsersActiveTheme = (userId: string) => {
+    return this._apiClient.getUsersActiveThemeAsync(userId);
+  };
+
+  loadThemeForUser = (userId: string) => {
+    if (isNullOrWhitespace(userId)) return;
+
+    this._apiClient.getAllThemesAsync().then((themes) => {
+      this._apiClient
```

Comment:
Unlikely but try/catch here would be good and for other async calls (saw some in the previous files)

Reply:
removed the method as it was not used

Reply:
in terms of other try/catch I agree but more at the top level, essentially generally I'd only try/catch where you can do something about the problem that might occur. Can talk this through further though if you like?

Reply:
Agree, makes sense